azure-pipelines.yml

```diff
@@ -150,7 +150,7 @@ stages:
       - task: UseDotNet@2
         inputs:
           packageType: sdk
-          version: 5.0.100-preview.7.20366.6
+          version: 5.0.100-preview.8.20417.9
```

Comment:
What is the motivation to do this now? Regarding the sideeffects on peachee jobs that take more time to finish.

Reply:
Preview 8 was released and it's now feature complete. This makes preview 7 obsolete. The main idea is to have early feedback.

Reply:
Side effects: To fully move to preview 8, we need to reconfigure/rebuild peachee VM to install that SDK.

Reply:
Indeed, this is just the fist step. I will update the image and the build script as well.